scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/context/TaskContextVariableExtractor.java

```diff
@@ -350,4 +374,25 @@
         }
         return variables;
     }
+
+    /**
+     * Extract job map results from the previous task result to be used now.
+     *
+     * @param taskContext contains the information needed to extract.
+     *
+     * @return a map containing extracted job results variables or an empty hash if there are no variables.
+     */
+    private Map<String, Serializable> extractJobMapVariables(TaskContext taskContext)
+            throws IOException, ClassNotFoundException {
+        Map<String, Serializable> JobMap = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)